benchmark/torch/maddpg/train.py

```diff
@@ -19,15 +19,15 @@
 from simple_model import MAModel
 from simple_agent import MAAgent
 from parl.algorithms import MADDPG
-from parl.env.multiagent_simple_env import MAenv
+from parl.env.pettingzoo_mpe import MAenv_v2
```

Comment:
Can we rename the env path in PARL:
`from parl.env.multiagent_env import env ` ?
The user does not care the env version.

Reply:
What about renaming with `from parl.env.multiagent_env import MAenv`. 
Using `import env` may be inconvenient to set the variable name `env` when developing train.py.